.vsts.pipelines/phases/ci-linux.yml

```diff
@@ -129,9 +129,13 @@ phases:
 
   # tar the tarball directory into the drop directory.
   - script: |
-      $(docker.run) $(docker.tb.map) $(docker.drop.map) $(imageName) /bin/bash -c '
+      $(docker.run) $(docker.tb.map) $(docker.drop.map) $(docker.tb.work) $(imageName) /bin/bash -c '
         mkdir -p /drop/tarball/
-        tar -zcf "/drop/tarball/$(tarballName).tar.gz" "/tb/$(tarballName)"'
+        smokeTestPackages="$(tarballName)/prebuilt/smoke-test-packages"
+        find "$smokeTestPackages" -iname "*.nupkg" -exec mv {} "$smokeTestPackages" \;
```

Comment:
What is this line doing?  It looks like it's finding all *.nupkg files and moving them into the same directory it found them in.  Am I missing something?

Reply:
Package cache structure is like this:

```
├── commandlineparser
│   └── 2.1.1-beta
│       ├── commandlineparser.2.1.1-beta.nupkg
... (contents of the nupkg)
├── coveralls.io
│   └── 1.4.0
│       ├── coveralls.io.1.4.0.nupkg
...
```

These `find`s put all the nupkgs into the root folder (like they are in `prebuilt/nuget-packages`), and cleans up the expanded files.

Reply:
Oh, got it.  Thanks.

Reply:
I'll add a comment. Just the `-delete` would probably produce a usable tarball (since NuGet does search recursively) but it seems better to keep the structure the same as the other prebuilt dir.